samples/excel/01-basics/api-call-with-notification-toast.yaml

```diff
@@ -48,32 +46,21 @@ script:
         }
     language: typescript
 template:
-    content: |
-        <!-- Fabric Message Banner template -->
-        <!-- https://dev.office.com/fabric-js/Components/MessageBanner/MessageBanner.html -->
-        <div class="ms-MessageBanner">
-          <div class="ms-MessageBanner-content">
-            <div class="ms-MessageBanner-text">
-              <div class="ms-MessageBanner-clipper"></div>
-            </div>
-            <button class="ms-MessageBanner-expand">
-              <i class="ms-Icon ms-Icon--ChevronDown"></i>
-            </button>
-          </div>
-          <button class="ms-MessageBanner-close">
-            <i class="ms-Icon ms-Icon--Clear"></i>
-          </button>
-        </div>
-
-        <section class="ms-font-m">
-            <p>Color the selected range in yellow, and display the selection address in a notification toast.</p>
-            <button id="run" class="ms-Button">
-                <span class="ms-Button-label">Run code</span>
-            </button>
-        </section>
+    content: "<!-- Fabric Message Banner template -->\n<!-- https://dev.office.com/fabric-js/Components/MessageBanner/MessageBanner.html -->\n<div class=\"ms-MessageBanner\">\n\t<div class=\"ms-MessageBanner-content\">\n\t\t<div class=\"ms-MessageBanner-text\">\n\t\t\t<div class=\"ms-MessageBanner-clipper\"></div>\n\t\t</div>\n\t</div>\n\t<button class=\"ms-MessageBanner-close\">\n    <i class=\"ms-Icon ms-Icon--Clear\"></i>\n  </button>\n</div>\n\n<section id=\"main\" class=\"ms-font-m\">\n\t<p>Color the selected range in yellow, and display the selection address in a notification toast.</p>\n\t<button id=\"run\" class=\"ms-Button\">\n        <span class=\"ms-Button-label\">Run code</span>\n    </button>\n</section>"
```

Comment:
It's doesn't help, newline or not.  No idea what is causing the YAML engine to want to format one way or another, but it's not the newline...